src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
```

Comment:
```suggestion
    private List<Long> ticketIds;
```
Please fix the same issue in all places